blueprints/component/files/__root__/__path__/__name__.ts

```diff
@@ -1,7 +1,4 @@
 import Component from '@ember/component';
 <%= importTemplate %>
-export default class <%= classifiedModuleName %> extends Component.extend({
-  // anything which *must* be merged to prototype here
-}) {<%= contents %>
-  // normal class body definition here
+export default class <%= classifiedModuleName %> extends Component {
```

Comment:
Since the comment `// class body definition here` is kept in the file `blueprints/component/glimmer-files/__root__/__path__/__name__.ts`, should we keep it as well in this file?

Reply:
Nope, we want all of those gone!

Reply:
(I'm working on a PR that will follow on from #216 to remove it from all the *other* definitions as well.)

Reply:
That's what I suspected. I knew I read a "let's remove these comments" somewhere but couldn't remember where.

Reply:
Let's do the opposite and remove the comment from the file `blueprints/component/glimmer-files/__root__/__path__/__name__.ts` then.